### PR TITLE
Update @orb.yml

### PR DESCRIPTION
**Pull request by PhilipAbed**
sync orb version 20.3.1

### CODE REVIEW

File: .circleci/orbs/orb-fsa/@orb.yml [hunk 1]
```diff
@@ -17,21 +18,33 @@ jobs:
...
+        default: "${API_KEY}"
       directory:
         description: Comma separated list of directories and / or files to scan.
         type: string
         default: "."
       config_file_path:
         description: Configuration file name (including file path).
         type: string
-        default: "./whitesource-fs-agent.config"
+        default: "whitesource-fs-agent.config"
```
Comment (nabeelsaabna) [inline .circleci/orbs/orb-fsa/@orb.yml:29]: Should this value be updated to `wss-unified-agent.config` ?

File: .circleci/orbs/orb-fsa/@orb.yml [hunk 2]
```diff
@@ -1,14 +1,15 @@
 version: 2.1
-description: "WhiteSource Unified Agent is a tool which extracts descriptive information from your open source libraries located on your file system."
+description: "The orb integrates WhiteSource Unified Agent, a tool which extracts descriptive information from your open source libraries located on your file system. For more information: https://whitesource.atlassian.net/wiki/spaces/WD/pages/33718339/Unified+Agent"
```
Comment (LenaKleyner) [inline .circleci/orbs/orb-fsa/@orb.yml]: Please refine to:
description: "The orb integrates WhiteSource Unified Agent, a tool for detecting security vulnerabilities and compliance data related to open source components. For more information: https://whitesource.atlassian.net/wiki/spaces/WD/pages/804814917/Unified+Agent+Overview"

File: .circleci/orbs/orb-fsa/@orb.yml (hunk 1 above)
Comment (PhilipAbed) [reply .circleci/orbs/orb-fsa/@orb.yml:29]: This is tested.. the old one wasn't properly tested ( was done by someone else)

File: .circleci/orbs/orb-fsa/@orb.yml (hunk 2 above)
Comment (PhilipAbed) [reply .circleci/orbs/orb-fsa/@orb.yml:2]:
```suggestion
description: "The orb integrates WhiteSource Unified Agent, a tool which extracts descriptive information from your open source libraries located on your file system. For more information: https://whitesource.atlassian.net/wiki/spaces/WD/pages/33718339/Unified+Agent"
```

Comment (PhilipAbed) [reply .circleci/orbs/orb-fsa/@orb.yml]:
```suggestion
description: "The orb integrates WhiteSource Unified Agent, a tool for detecting security vulnerabilities and compliance data related to open source components. For more information: https://whitesource.atlassian.net/wiki/spaces/WD/pages/804814917/Unified+Agent+Overview"
```

**Review by nabeelsaabna — APPROVED** [review]

**Pull request merged**